### PR TITLE
feat(module: button): color

### PR DESCRIPTION
**Pull request by boukenka**
### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

[Button color](https://github.com/ant-design-blazor/ant-design-blazor/issues/1761)

### 💡 Background and solution

Here is an example : 

Example

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Allows you to set a color for the button according to the official palette.       |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1774) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1774) before we can accept your contribution.<br/><br/>:x: boukenka [sign now](https://cla.dotnetfoundation.org/ant-design-blazor/ant-design-blazor?pullRequest=1774)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/ant-design-blazor/ant-design-blazor?pullRequest=1774) it.</sub>

File: components/button/Button.razor.cs [hunk 1]
```diff
@@ -174,5 +181,143 @@ private async Task OnMouseUp(MouseEventArgs args)
                 await InvokeAsync(StateHasChanged);
             });
         }
+
+        private string SetButtonColorStyle() => Color switch
+        {
+            ButtonColor.Red1 => Style = "background-color: #fff1f0; border-color: #fff1f0; color: rgba(0,0,0,0.85);",
```
Comment (ElderJames) [inline components/button/Button.razor.cs:187]: Where did you find these colors?

Comment (boukenka) [reply components/button/Button.razor.cs:187]: They are from here: [Colors](https://ant.design/docs/spec/colors).

Comment (ElderJames) [reply components/button/Button.razor.cs:187]: It's awesome.

Comment (ElderJames) [reply components/button/Button.razor.cs:187]: Please put them into a static Dictionary.

**Comment by ElderJames** [comment]
Please switch the target branch to `feature`.

**Comment by boukenka** [comment]
> Please switch the target branch to feature.

Done.

File: components/button/Button.razor.cs (hunk 1 above)
Comment (boukenka) [reply components/button/Button.razor.cs]: Thank you. Done.

**Comment by ElderJames** [comment]
Also need to remove other's commit

image

**Comment by boukenka** [comment]
> Also need to remove other's commit

I did the following. Not sure this is correct. 

```
git reset --hard e8ec6ea
git reset --hard 667120d
git reset --hard bdcc42c
git reset --hard 015129c
git reset --hard cbe94d1
git push -f origin button-color
```

**Comment by ElderJames** [comment]
Oh, no. You remove your correct commit...

**Comment by boukenka** [comment]
Let's delete this PR and I will create another one directly from the _feature_ branch?

**Comment by ElderJames** [comment]
You need to rebase on feature branch. Than commit your changes again, and push to currently branch.

**Review by ElderJames — APPROVED** [review]

**Pull request merged**